.github/workflows/reusable_dispatch_analytics.yml

```diff
@@ -3,10 +3,40 @@ name: Reusable dispatch analytics
 on:
   workflow_dispatch:
 
+  schedule:
+    - cron: '0 * * * *'
+
+permissions:
+  id-token: write
+  actions: read
+  checks: read
+  contents: read
+  deployments: read
+  issues: read
+  discussions: read
+  packages: read
+  pages: read
+  pull-requests: read
+  repository-projects: read
+  security-events: read
+  statuses: read
+
 jobs:
```

Comment:
Can we run in an environment do secrets are encrypted from forks?

You never know if we ever accidentally make a mistake in the IAM OIDC and allow any repo. This will prevent this attack vector.

Reply:
Done, created a new "analytics" environment and moved the secrets there. I've removed the old secrets too.

Reply:
Please note that the IAM OIDC would not be assumable since there's a condition to only be assumable by this repository too. In any case, it's better this way.